include/SFML/System/Time.hpp

```diff
@@ -78,6 +87,23 @@ class Time
     ////////////////////////////////////////////////////////////
     constexpr Int64 asMicroseconds() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Return the time value as a std::chorono::duration
+    ///
+    /// \return Time in microseconds
+    ///
+    ////////////////////////////////////////////////////////////
+    constexpr std::chrono::microseconds toDuration() const;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Implicit conversion to std::chrono::duration
+    ///
+    /// \return Duration in microseconds
+    ///
+    ////////////////////////////////////////////////////////////
+    template <typename Rep, typename Period>
+    [[nodiscard]] constexpr operator std::chrono::duration<Rep, Period>() const;
```

Comment:
Out of curiosity regarding `[[nodiscard]]`: is it even possible to invoke this operator _without_ using the result? Since its whole point is to be called when `sf::Time` is used in a context that would allow the conversion.

That is, apart from the academic `time.operator std::chrono::duration<Rep, Period>()`.

Reply:
I can't think of any such scenario.

Reply:
Then maybe remove it, as it's more confusing to the code reader if it doesn't have a real purpose?

Reply:
I don't think it would ever result in a net negative to the user. I much prefer liberally applying `[[nodiscard]]` but I know you prefer omitting it in many instances so I'll get rid of it here.

Reply:
It's more a case of reducing clutter where it doesn't add useful information. If there are fewer instances of `[[nodiscard]]`, people will pay more attention to them and understand that those are potential pitfalls.